cmd/launcher/main.go

```diff
@@ -45,6 +46,13 @@ func runMain() int {
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 
+	// these systemslogger logs will go to stderr, which will dirty the results of our
+	// rundisclaimed operations. the only log we would see before running the subcommand
+	// should be "launcher starting up", so override here with a no-op logger
+	if len(os.Args) > 1 && os.Args[1] == "rundisclaimed" {
```

Comment:
Maybe we would want to formalize the subcommands that should run "silently" because we are expecting a particular output? Since technically `query-windowsupdates` would also need this, we just got lucky that the systemSlogger writes to the event log instead of stdout. I don't think it's worth holding up this PR for, but if it seems like a good idea, maybe we file an issue to tackle it later?

Reply:
yeah are there others you think would benefit? i.e. worth a full audit in separate issue or should I just add `query-windowsupdates` to the check here?

Reply:
That was the only one where I thought it'd actually matter -- but maybe it would be less confusing to end users if we also didn't print all these extra logs during `version`, `flare`, `doctor`?

Reply:
discussed this more [here](https://github.com/kolide/launcher/pull/2196/files#r2035680933) - I think we're going to remove this and encourage callers to handle stdout/stderr separately, this was only an issue for brew because we were using the same buffer for both. let me know if you still think worth adding an issue for flare/doctor? I could go either way

Reply:
Nah I think this is fine as-is!